easybuild/easyconfigs/g/GPAW/GPAW-22.8.0-intel-2022a.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '22.8.0'
+
+homepage = 'https://wiki.fysik.dtu.dk/gpaw/'
+description = """GPAW is a density-functional theory (DFT) Python code based on the projector-augmented wave (PAW)
+ method and the atomic simulation environment (ASE). It uses real-space uniform grids and multigrid methods or
+ atom-centered basis-functions."""
+
+toolchain = {'name': 'intel', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'openmp': False}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    ('GPAW-20.1.0-Add-Easybuild-configuration-files.patch', 1),
+    ('GPAW-22.8.0-Remove-MPI-compiler-hack.patch', 0),
+]
+checksums = [
+    '311eae8cb046f720e77e2e955042758c6e66d37c1fcbaf5d35e2834294a4ed6c',  # gpaw-22.8.0.tar.gz
+    # GPAW-20.1.0-Add-Easybuild-configuration-files.patch
+    '2b337399479bf018a86156ed073dd7a78ec8c0df1f28b015f9284c6bf9fa5f15',
+    '44ca47fefb1bd409bae7da0a2f35125571dc927dc98f7cbca948ce86d25aa77e',  # GPAW-22.8.0-Remove-MPI-compiler-hack.patch
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('ASE', '3.22.1'),
+    ('libxc', '5.2.3'),
```

Comment:
is `libvdwxc` missing here?

Reply:
Unfortunately, ``libvdwxc`` does not work with the Intel compilers.  

The developer @askhl tells me it is because it uses parts of FFTW not reimplemented by MKL, and linking with FFTW fails because MKL redefines some symbols used internally by these FFTW functions, but with a slightly different API.  The end result is that GPAW core dumps if it tries to call libvdwxc when built with Intel compilers.  I have given up on finding a workaround.